magento/pwa-studio
# [PWA-1003] Increase Test Coverage in venia-ui/lib/components/CreateAccountPage

## Description

Increase test coverage for the CreateAccountPage component.

## Related Issue





Closes #PWA-1003.

## Acceptance



Any developer
### Verification Stakeholders

### Specification


Any developer
### Verification Steps


1. Run `yarn test --collectCoverageFrom=**/createAccountPage.js packages/venia-ui/lib/components/CreateAccountPage/`
2. Verify tests succeed
3. Verify acceptable test coverage

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
